src/vm/ceeload.h

```diff
@@ -3686,6 +3686,7 @@ class ReflectionModule : public Module
     // If true, then only other transient modules can depend on this module.
     bool m_fIsTransient;
 
+#if !defined DACCESS_COMPILE && !defined CROSSGEN_COMPILE
```

Comment:
Correct on the de-optimization. Most of these changes make the .h file match the .cpp with respect to #ifdefs. For example, ceeload.h defines ReflectionModule::IsMetadataCaptureSuppressed() but the .cpp has #ifdefs for DACCESS_COMPILE and CROSSGEN_COMPILE. So the change shown above adds the same #ifdef to the .h.